Firmware/printer.cfg

```diff
@@ -34,22 +57,23 @@ uart_pin: PC11
 tx_pin: PC10
 uart_address: 0
 interpolate: True
-run_current: 0.5        # For V0.1 spec NEMA14 40Ncm
+## LDO Motor
+run_current: 0.5
 hold_current: 0.25
 sense_resistor: 0.110
-stealthchop_threshold: 0
-
+stealthchop_threshold: 0					# Set to 9999 to turn off stealthchop
 
 [stepper_y]
 step_pin: PB10
-dir_pin: PB2            # check https://docs.vorondesign.com/build/startup/#v0 if moving in the wrong direction
+dir_pin: PB2								# Add ! if moving opposite direction
```

Comment:
this tip needs to get users to the homing direction graphic somehow because "wrong" direction for a single motor in corexy doesnt necessarily mean adding/removing the !